cargo-miri/bin.rs

```diff
@@ -34,10 +35,11 @@ Examples:
     cargo miri test -- test-suite-filter
 "#;
 
-#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 enum MiriCommand {
     Run,
     Test,
+    Nextest(String),
```

Comment:
Honestly all of these sound like they could just `Cargo(String)`, since all the handling after the initial parsing seems to be uniform? (only `MiriCommand::Setup` seems to be special)

<hr/>

Could even support `nextest` "by omission", removing this error:
```
`cargo miri` supports the following subcommands: `run`, `test`, `nextest`, and `setup`.
```

OTOH it would be weird to have e.g. `cargo miri update` and other stuff work, heh.

Reply:
Oops, #2399 already did that, nevermind.